<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetFolders">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Project Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidgetProjectFolder">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>Project Structure</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Raw Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidgetDataFolder">
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>DataFolder Structure</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <widget class="QMenu" name="menuAdd_To_Project">
     <property name="title">
      <string>Add To Project</string>
     </property>
     <addaction name="actionScrape_New_Data"/>
     <addaction name="actionLoad_Existing_Dataset"/>
    </widget>
    <widget class="QMenu" name="menuProject_2">
     <property name="title">
      <string>Project...</string>
     </property>
     <addaction name="actionNew_Project"/>
     <addaction name="actionLoad_Project"/>
    </widget>
    <addaction name="menuProject_2"/>
    <addaction name="menuAdd_To_Project"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuCalculate">
     <property name="title">
      <string>Calculate...</string>
     </property>
     <addaction name="actionParameters_and_Metadata_for_all"/>
     <addaction name="actionParameters_for_selection"/>
     <addaction name="actionMissing_Parameter"/>
     <addaction name="separator"/>
     <addaction name="actionOnly_Metadata"/>
    </widget>
    <widget class="QMenu" name="menuCenterfinding">
     <property name="title">
      <string>Centerfinding</string>
     </property>
     <addaction name="actionCenterfinding_All"/>
     <addaction name="actionCenterfinding_selection"/>
    </widget>
    <addaction name="menuCenterfinding"/>
    <addaction name="menuCalculate"/>
    <addaction name="actionOpen_Data_Explorer"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScrape_Data">
   <property name="text">
    <string>Scrape Data</string>
   </property>
  </action>
  <action name="actionLoad_Pickle">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionDetect_Circles">
   <property name="text">
    <string>Detect Circles</string>
   </property>
  </action>
  <action name="actionAdd_to_Prioject">
   <property name="text">
    <string>Add to Prioject</string>
   </property>
  </action>
  <action name="actionScrape_New_Data">
   <property name="text">
    <string>Scrape New Data</string>
   </property>
  </action>
  <action name="actionLoad_Existing_Dataset">
   <property name="text">
    <string>Load Existing Dataset</string>
   </property>
  </action>
  <action name="actionCalculate_Common_Metadata">
   <property name="text">
    <string>Calculate only Metadata</string>
   </property>
  </action>
  <action name="actionbanana">
   <property name="text">
    <string>banana</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="actionParameters_and_Metadata_for_all">
   <property name="text">
    <string>Parameters for all</string>
   </property>
  </action>
  <action name="actionParameters_for_selection">
   <property name="text">
    <string>Parameters for selection</string>
   </property>
  </action>
  <action name="actionOnly_Metadata">
   <property name="text">
    <string>Only Metadata</string>
   </property>
  </action>
  <action name="actionCenterfinding_All">
   <property name="text">
    <string>Centerfinding All</string>
   </property>
  </action>
  <action name="actionCenterfinding_selection">
   <property name="text">
    <string>Centerfinding Selection</string>
   </property>
  </action>
  <action name="actionMissing_Parameter">
   <property name="text">
    <string>Missing Parameters</string>
   </property>
  </action>
  <action name="actionOpen_Data_Explorer">
   <property name="text">
    <string>Open Data Explorer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
